src/containers/List/ListFilters/common.tsx

```diff
@@ -75,7 +80,7 @@ const FilterOptions: React.FC<FilterOptionsProps> = ({
                 Status?.[option as ApplicationStatus] ??
                 AssignAction?.[option as AssignerAction] ??
                 ReviewAction?.[option as ReviewerAction] ??
-                startCase(option)
+                option
```

Comment:
I think Andrei had `startCase` on when we were displaying the Enum values directly from the back-end, which are ALL_CAPS. However, we now map these to localised strings, so "startCase" is not necessary (and makes a mess of values that SHOULD be uppercase, such as registration numbers).